test/new_relic/agent/serverless_handler_test.rb

```diff
@@ -276,10 +276,10 @@ def test_custom_attributes_arent_supported_when_serverless
         skip_unless_minitest5_or_above
 
         attrs = {cool_id: 'James', server: 'less', current_time: Time.now.to_s}
+        attribute_set_attempted = false
         tl_current_mock = Minitest::Mock.new
         tl_current_mock.expect :add_custom_attributes, -> { attribute_set_attempted = true }, [attrs]
```

Comment:
The warning for this one said `attribute_set_attempted` was assigned, but never used, with the `mock.expect` line as the culprit. Defining the argument before the proc removed the warning.